packages/babel-plugin-transform-block-scoping/test/fixtures/general/issue-8128-for-of-after/output.js

```diff
@@ -0,0 +1,22 @@
+var _loop = function (_result) {
+  _result = otherValue;
+  fn(() => {
+    _result;
+  });
+  result = _result;
+};
+
+var _iterator = babelHelpers.createForOfIteratorHelper(results),
+    _step;
+
+try {
+  for (_iterator.s(); !(_step = _iterator.n()).done;) {
+    var result = _step.value;
```

Comment:
Plugin ordering changed the output, but that's unavoidable since both apply different transformations to the same thing.